applications/projects/runSofa/runSofa_test/runSofa_test.cpp

```diff
@@ -0,0 +1,84 @@
+/******************************************************************************
+*       SOFA, Simulation Open-Framework Architecture, development version     *
+*                (c) 2006-2017 INRIA, USTL, UJF, CNRS, MGH                    *
+*                                                                             *
+* This program is free software; you can redistribute it and/or modify it     *
+* under the terms of the GNU General Public License as published by the Free  *
+* Software Foundation; either version 2 of the License, or (at your option)   *
+* any later version.                                                          *
+*                                                                             *
+* This program is distributed in the hope that it will be useful, but WITHOUT *
+* ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or       *
+* FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for    *
+* more details.                                                               *
+*                                                                             *
+* You should have received a copy of the GNU General Public License along     *
+* with this program. If not, see <http://www.gnu.org/licenses/>.              *
+*******************************************************************************
+* Authors: The SOFA Team and external contributors (see Authors.txt)          *
+*                                                                             *
+* Contact information: contact@sofa-framework.org                             *
+******************************************************************************/
+
+#include <fstream>
+#include <gtest/gtest.h>
+#include <SofaTest/Sofa_test.h>
+
+#include <sofa/helper/Utils.h>
+#include <sofa/helper/Utils.h>
+#include <sofa/helper/system/PluginManager.h>
+#include <sofa/helper/system/FileRepository.h>
+
+namespace sofa
+{
+
+class runSofa_test : public Sofa_test<>
+{
+protected:
+    std::string m_testConfigPluginName;
+    std::string m_testConfigPluginPath;
+    std::string m_testPluginName;
+
+    runSofa_test() {
+
+    }
+
+    void SetUp()
+    {
+#ifdef WIN32
```

Comment:
Could we have a function in sofa::helper::Utils::getPluginDir() to avoid having #ifdef in the main application ?